integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/InfinispanStoreJdbcIT.java

```diff
@@ -32,7 +34,10 @@
 
    @Deployment
    public static Archive<?> deployment() {
-      return ShrinkWrap.create(WebArchive.class, "jdbc.war").addClass(InfinispanStoreJdbcIT.class).add(manifest(), "META-INF/MANIFEST.MF");
+      return ShrinkWrap.create(WebArchive.class, "jdbc.war")
+            .addClass(InfinispanStoreJdbcIT.class)
+            .addAsResource("jdbc-config.xml")
+            .add(manifest(), "META-INF/MANIFEST.MF");
```

Comment:
This test should pass even w/o explicitly declaring a dependency on the `org.infinispan.persistence.jdbc` module.

One shouldn't repeat himself by having to enable the CacheStore explicitly in two different places:
 - the configuration file
 - the module structure
 - (and a third!) having the depenency

Steps 1 and 3 are widely accepted, so hardly surprising. Step 2 should be handled automatically by Infinispan.

I appreciate you might disagree, but in that case at least throw a better error message pointing out what's missing rather than a generic parser exception.